includes/admin/class-wcs-admin-system-status.php

```diff
@@ -124,6 +125,19 @@ private static function set_live_site_url( &$debug_data ) {
 		);
 	}
 
+	/**
+	 * @param array $debug_data
+	 */
+	private static function set_library_version( &$debug_data ) {
+		$debug_data['wcs_subs_core_version'] = array(
+			'name'      => _x( 'Subscriptions Library Version', 'Subscriptions-Core Version, Label on WooCommerce -> System Status page', 'woocommerce-subscriptions' ),
+			'label'     => 'Subscriptions Library Version',
+			'note'      => WC_Subscriptions_Core_Plugin::instance()->get_plugin_version(),
```

Comment:
Should this be `get_library_version()` since `get_plugin_version()` is deprecated?

https://github.com/Automattic/woocommerce-subscriptions-core/blob/d08ed6617ad73fb8dc3dd16bce23a4817ae5c78c/includes/class-wc-subscriptions-core-plugin.php#L258-L266